js/src/ast/expressions/FuncCall.ts

```diff
@@ -51,49 +51,26 @@ export class FuncCall extends Base implements Expression, Statement {
     })
     if (funcExpr.type) {
       const resolvedFuncType = resolve(funcExpr.type)
+      let returnType: NType | null = resolvedFuncType
       if (resolvedFuncType instanceof FuncType) {
-        let returnType: NType = resolvedFuncType
```

Comment:
Apparently if it was just `let returnType: NType | null ...` inside the if statement, TypeScript gets pissed 😔